packages/patternfly-4/src/components/switcher/index.js

```diff
@@ -8,21 +9,30 @@ import { Link } from 'gatsby';
 import { Location } from '@reach/router';
 import './styles.scss';
 
-const Switcher = () => (
+const trimTrailingSlash = str => str.replace(/\/$/, '');
+const verifyPath = (path, pages) =>
+  pages.edges
+    .map(edge => trimTrailingSlash(edge.node.path))
+    .find(queryPath => queryPath === trimTrailingSlash(path));
```

Comment:
Won't this not work for aboutmodal vs aboutmodalbox? Maybe compare up to the shorter of the two?

Reply:
I think ideally we would want to rename one or the other of those two to make them consistent.

Reply:
Conveniently enough for now, those two happen to be the defaults, so toggling between them will still work despite not matching at all :)

Yes, I'm a little ashamed of this.